src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.Core.targets

```diff
@@ -44,6 +44,16 @@ It is shared between "legacy" binding projects and .NET 5 projects.
     </PropertyGroup>
   </Target>
 
+  <Target Name="_IncrementalCleanBindings"
+    Condition="Exists ('$(GeneratedOutputPath)') And '@(InputJar->Count())' == '0' And '@(EmbeddedJar->Count())' == '0' And '@(LibraryProjectZip->Count())' == '0' And '@(_JavaBindingSource->Count())' == '0' "
+  >
+    <PropertyGroup>
+      <_AndroidBindingIncrementalClean>True</_AndroidBindingIncrementalClean>
+    </PropertyGroup>
+    <CallTarget Targets="CleanBindingsOutput" />
```

Comment:
Instead of manually cleaning, is there a way to fix the inputs of the other targets, so they would just run again?

Or does this fix a different issue?